lib/rules/require-ember-lifeline.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileOverview Require the use of ember-lifeline's runTask over using Ember.run.later.
+ * @author Steve Calvert
```

Comment:
But how will the world know about my amazing feats of coding? 

Totally agree. I'll remove from all rules in a separate PR.